plugins/SitesManager/SiteContentDetection/Cloudflare.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link    https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\Plugins\SitesManager\SiteContentDetection;
+
+use Piwik\Piwik;
+use Piwik\Request;
+use Piwik\SiteContentDetector;
+use Piwik\View;
+
+class Cloudflare extends SiteContentDetectionAbstract
+{
+    public static function getName(): string
+    {
+        return 'Cloudflare';
+    }
+
+    public static function getContentType(): string
+    {
+        return self::TYPE_CMS;
```

Comment:
We could also use OTHER I guess. I've put it into CMS, as it was detected as CMS before as well. Not sure if changing that would cause problems, as the checks for detected cms wouldn't succeed for Cloudflare anymore.
Might be something we could change after the new layout was implemented...